src/librustc/metadata/common.rs

```diff
@@ -139,14 +139,15 @@ pub enum astencode_tag { // Reserves 0x40 -- 0x5f
     tag_table_adjustments = 0x51,
     tag_table_moves_map = 0x52,
     tag_table_capture_map = 0x53,
-    tag_table_closures = 0x54,
+    tag_table_closure_tys = 0x54,
```

Comment:
I mean, add the second one after the first one, to avoid changing *the name* of the last tag in the list. The values don't really matter as long as they don't overlap with anything else.